src/plugins/gui/RewardsCardPlugin.tsx

```diff
@@ -106,19 +105,26 @@ export const makeRewardsCardPlugin: FiatPluginFactory = async params => {
     })
   }
 
-  const showDeleteItemModal = async (item: RewardsCardItem) => {
+  const showDeleteItemModal = async (card: RewardsCardItem) => {
     const answer = await Airship.show<'delete' | 'keep' | undefined>(bridge => (
       <ButtonsModal
         bridge={bridge}
         buttons={{
-          keep: { label: lstrings.string_keep, type: 'primary' },
-          delete: { label: lstrings.string_delete, type: 'escape' }
+          delete: { label: lstrings.string_delete, type: 'secondary' },
+          keep: { label: lstrings.string_keep, type: 'escape' }
         }}
         title={lstrings.rewards_card_delete_modal_title}
-        message={sprintf(lstrings.rewards_card_delete_modal_message_s, item.expiration)}
+        message={sprintf(lstrings.rewards_card_delete_modal_message_s, card.expiration)}
```

Comment:
While still imperfect, the text could be a bit clearer like "Are you sure you want to delete the Visa Card with the following expiration?”

Or even "Are you sure you want to delete the Visa Card that expires %s?”

Either way, format the expiration with toLocaleString() as you did in the card itself